XPNet.GraphicsTest/Properties/PublishProfiles/FolderProfile.pubxml

```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this actually used for anything?  The GraphicsTest project isn't published on its own is it?

Reply:
Removed properties folder, because it is not needed